src/main/resources/static/scripts/register.js

```diff
@@ -8,16 +8,19 @@ function register() {
     var url = 'http://localhost:8080/auth/signup';
     var data = {"email": emailElement.value, "name": nameElement.value, "password": passwordElement.value};
 
-    if (passwordElement.value != repasswordElement.value){
+    if (passwordElement.value !== repasswordElement.value){
         document.getElementById("error").innerHTML = "Not the same passwords";
```

Comment:
If element is used more than once, assign it to variable.

Reply:
Maybe I should create something like this
    ```
let emailElement=document.getElementById("emailInput").value;
    let nameElement=document.getElementById("nameInput").value;
    let passwordElement=document.getElementById("passInput").value;
    let repasswordElement=document.getElementById("repassInput").value;
```

Reply:
Good idea, but call it just email or emailValue instead of emailElement